### ISSUE TITLE
apmhttp: panic with disabled tracer or ignored request

### ISSUE THREAD

**Issue by lobotomist** [elastic/apm-agent-go#612]
Seems potential bug:
https://github.com/elastic/apm-agent-go/blob/master/module/apmhttp/handler.go#L71

```
if !h.tracer.Active() || h.requestIgnorer(req) {
	h.handler.ServeHTTP(w, req)
	return
}
```

If the tracer is inactive or request has been ignored, in case of panic recovery step will be skipped as well.  This can be a problem for service which does not use additional recovery middleware.

**Comment by axw** [comment]
Yes, fair enough. The recovery process should continue even if the tracer is inactive, or the request is ignored.
